pages/_document/index.js

```diff
@@ -37,7 +38,6 @@ export default class MyDocument extends Document {
         </Head>
         <body>
           <Main />
-          <NextScript />
```

Comment:
I have no idea what this is for, but nothing seems to break when I remove it and it fixes the title issue so I think we're ok.